Formula/jhead.rb

```diff
@@ -1,13 +1,8 @@
 class Jhead < Formula
   desc "Extract Digicam setting info from EXIF JPEG headers"
-  homepage "https://www.sentex.net/~mwandel/jhead/"
-  url "https://www.sentex.net/~mwandel/jhead/jhead-3.04.tar.gz"
-  sha256 "ef89bbcf4f6c25ed88088cf242a47a6aedfff4f08cc7dc205bf3e2c0f10a03c9"
-
-  livecheck do
-    url :homepage
-    regex(/href=.*?jhead[._-]v?(\d+(?:\.\d+)+)\.t/i)
-  end
+  homepage "https://github.com/Matthias-Wandel/jhead"
+  url "https://github.com/Matthias-Wandel/jhead/archive/3.06.0.1.tar.gz"
+  sha256 "5c5258c3d7a840bf831e22174e4a24cb1de3baf442f7cb73d5ab31b4ae0b0058"
 
```

Comment:
```suggestion
  license :public_domain

```

From [readme](https://github.com/Matthias-Wandel/jhead/tree/3.06.0.1#readme):

> License:
> 
>     Jhead is public domain software - that is, you can do whatever you want
>     with it, and include it software that is licensed under the GNU or the
>     BSD license, or whatever other license you chose, including proprietary
>     closed source licenses.  Although not part of the license, I do expect
>     common courtesy, please.
> 
>     If you do integrate the code into some software of yours, I'd appreciate
>     knowing about it though.